canonical-web-and-design/maas-ui
# fix(ui): update label for kernel parameters in settings

## Done

- Updated label for kernel parameters in settings. "Boot parameters to pass to the kernel by default" => "Global boot parameters always passed to the kernel"
- Converted kernel parameters components to TS

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to settings > kernel parameters and check that the label has updated to say that the parameters are *always* passed

## Fixes

Fixes #2650
